themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/applications-page/ApplicationsPage.tsx

```diff
@@ -15,21 +15,196 @@
+                    id={'applicationToggle' + appIndex}
+                    aria-controls="expandable"
+                  />
+                  <DataListItemCells
+                    dataListCells={[
+                      <DataListCell width={2} key={'app-' + appIndex}>
+                        <BuilderImageIcon /> <Text component={TextVariants.small}>{application.clientId}</Text>
+                      </DataListCell>,
+                      <DataListCell width={2} key={'internal-' + appIndex}>
+                        <span>{application.inUse && !application.userConsentRequired ? 'Internal' : 'Third-party'}
+                          {application.offlineAccess ? ', Offline Access' : ''}
+                        </span>
+                      </DataListCell>,
+                      <DataListCell width={2} key={'status-' + appIndex}>
+                        <span>{application.inUse ? 'In use' : 'Not in use'}</span>
+                      </DataListCell>,
+                      <DataListCell width={4} key={'baseUrl-' + appIndex}>
+                        <a href={application.baseUrl}><LinkIcon /> {application.baseUrl}</a>
+                      </DataListCell>,
+                    ]}
+                  />
+                </DataListItemRow>
+                <DataListContent
+                  noPadding={false}
+                  aria-label="Application Details"
+                  id="expandable"
+                  isHidden={!this.state.isRowOpen[appIndex]}
+                >
+                  <Grid sm={12} md={12} lg={12}>
+                    <GridItem><b>{'Client: '}</b> {application.clientId}</GridItem>
+                    {application.description &&
+                      <GridItem><b>{'Description: '}</b> {application.description}</GridItem>
+                    }
+                    <GridItem><b>{'URL: '}</b> {application.baseUrl}</GridItem>
+                    {application.consent &&
+                      <React.Fragment>
+                        <GridItem span={12}>
+                          <b>Has access to:</b>
+                        </GridItem>
+                        {application.consent.grantedScopes.map((scope: GrantedScope, scopeIndex: number) => {
+                          return (
+                            <React.Fragment key={'scope-' + scopeIndex} >
+                              <GridItem offset={1}><CheckIcon /> {scope.name}</GridItem>
+                            </React.Fragment>
+                          )
+                        })}
+                        <GridItem><b>{'Access granted on: '}</b>
+                          {new Intl.DateTimeFormat('en-US', {
+                            year: 'numeric',
+                            month: 'long',
+                            day: 'numeric',
+                            hour: 'numeric',
+                            minute: 'numeric',
+                            second: 'numeric'
+                          }).format(application.consent.createDate)}
```

Comment:
This should be localized based on current user locale (incl. 12h/24h clock). But this is probably a bigger task (the same issue is at many different places in the Account Console) so it's probably out of scope of this PR.

Reply:
@vmuzikar agree, we can follow up on this in another Jira.

Reply:
Yes, I think we need a general JIRA for the whole app.  Let's start a JIRA and just add all the places we see that need l18n for time.

Reply:
Created a parent JIRA: https://issues.jboss.org/browse/KEYCLOAK-12338